msfs-avionics-mirror/src/sdk/components/FSComponent.ts

```diff
@@ -254,9 +254,23 @@ export namespace FSComponent {
     'rect': true,
     'stop': true,
     'svg': true,
-    'text': true
+    'text': true,
+    'tspan': true,
   };
 
+
+  const mappedKeys: Record<string, string> = {
+      'fontSize': 'font-size',
```

Comment:
Any particular reason not use the "real" keys instead of the mapped ones? Could be dangerous should we ever decide to update the framework (I know I started with modifying the files :D )

Reply:
Part of it was me being used to react, and for some reason thinking that the ones with dashes would not work.

I can revert them if the dashed ones work well.

Reply:
I don't think we should add this if the dashed ones are good. The react way was always ick but needed due to react.

Reply:
Changes removed in rebase